spec/operator/index_spec.lua

```diff
@@ -50,7 +50,7 @@ describe("[]", function()
          local e: Keys = "foo"
          local n: number = x[e]
       ]], {
-         { msg = "cannot index, not all enum values map to record fields of the same type" },
+         { msg = "cannot index, not all enum values map to record fields of the same type (types of fields" },
```

Comment:
is it possible to add the full message here with the field names? or is the check order non-deterministic? (I tried to catch all ordering issues a while ago; since the test runs over `ipairs` of the field names, they should return the same results every time)

Reply:
Indeed, I was not sure the order was deterministic. OK, I will add the full message.
(for the moment, I just rebased to see if the check was working)

Reply:
Done…